src/components/user/tour/create-tour/tour-detail.js

```diff
@@ -0,0 +1,141 @@
+import React, {Component} from 'react';
+import TextField from "@material-ui/core/TextField";
+import FormControl from '@material-ui/core/FormControl';
+import InputLabel from "@material-ui/core/InputLabel";
+import Input from "@material-ui/core/Input";
+
+export default class TourDetail extends Component {
+
+    constructor(props) {
```

Comment:
Please take a look at the terminal when you run the application. 
If have nothing to do in the constructor, there is no need to write a constructor.

```
src/components/user/tour/create-tour/tour-detail.js
  Line 9:5:  Useless constructor  no-useless-constructor
```